lms/djangoapps/verify_student/api.py

```diff
@@ -70,11 +76,20 @@ def create_verification_attempt(user: User, name: str, status: str, expiration_d
         expiration_datetime=expiration_datetime,
     )
 
+    emit_idv_attempt_created_event(
+        attempt_id=verification_attempt.id,
+        user=user,
+        status=status,
+        name=name,
+        expiration_date=expiration_datetime,
+    )
+
     return verification_attempt.id
 
 
 def update_verification_attempt(
     attempt_id: int,
+    user: User,
```

Comment:
I understand that you need this value to create the event object, but instead of modifying the function signature and requiring a parameter we don't really need, why don't we grab the `User` object off the `VerificationAttempt` instance? Otherwise, this is a breaking change.